VeroliBGV/BGV008.xml

```diff
@@ -0,0 +1,836 @@
+                        The handwriting is consistent with an 18th- or 19th-century dating.
+                     </origDate>
+                     <note/>
+                  </origin>
+                  <provenance>The manuscript originates from Eritrea according to <bibl>
+                     <ptr target="bm:Zanutto1932Bibliografia"/>
+                     <citedRange unit="page">92</citedRange>
+                  </bibl>). </provenance>
+               </history>
+               
+               <additional>
+                  <adminInfo>
+                     <recordHist>
+                        <source>
+                           <listBibl type="catalogue">
+                              <bibl>
+                                 <ptr target="bm:Zanutto1932Bibliografia"/>
+                                 <citedRange unit="page">93</citedRange>
+                                 <citedRange unit="item">8</citedRange>
+                              </bibl>
+                           </listBibl>
+                           <listBibl type="secondary">
+                              <bibl>
+                                 <ptr target="bm:Britaetc2017ThreeCollections"/>
+                                 <citedRange>178</citedRange>
+                              </bibl>
+                           </listBibl>
+                        </source>
+                     </recordHist>
+                  </adminInfo>
+               </additional>
+            </msDesc>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <projectDesc>
+            <p>Encoded according to TEI P5 Guidelines.</p>
+         </projectDesc>
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+            <xi:fallback>
+               <p>Definitions of prefixes used.</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc>
+         <textClass>
+            <keywords scheme="#ethioauthlist">
+               <term key="OldTestament"/>
+               <term key="Prayers"/>
+               <term key="Bible"/>
+               <term key="ChristianLiterature"/>
+            </keywords>
+         </textClass>
+         <langUsage>
+            <language ident="en">English</language>
+            <language ident="gez">Gǝʿǝz</language>
+         </langUsage>
+      </profileDesc>
+      <revisionDesc>
+         <change who="MV" when="2020-11-14">Created catalogue entry</change>
+         <change who="MV" when="2020-11-18">Textual description</change>
+         <change who="MV" when="2020-11-23">Encoded PhysDesc</change>
+         <change who="MV" when="2020-11-25">Added text edition</change>
+      </revisionDesc>
+  </teiHeader>
+   
+   <text xml:base="https://betamasaheft.eu/">
+      <body>
+         <div type="edition">
+            <div type="textpart" corresp="#i1.1" xml:lang="gez" xml:id="LIT4009Naahab">
```

Comment:
Massimo, sorry, can you please explain me what is this section? I do not know it.

Reply:
Anto, it is exactly, what I emat on the phone today. If we provide the ecomplete text of a work (e.g., short prayers) or like here of a section or of a work, we have to provide the text here. unfortuantely, it is not possible to have it under Ms-Item. there we have only incipit and explicit.
In the later visualisation, it will be placed to the content. Only for the encoding, we have to do it here. I also find it utmost inconvenient. If you have such a case, I can also explan it for you while you are encoding it.

Reply:
I see, Sanne. Thank you. Alright, I will ask you in case. Is this the case also when we have expanded titles, subscriptiones or colophons?

Reply:
Yes, You find the informatine here: https://betamasaheft.eu/Guidelines/?q=colophon&id=ColophSupplTit